_posts/2016-01-29-back-to-business.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Back to Business
+author: riikkanen
+excerpt: Turning from a mother to a software developer. A normal day for me.
+categories: 
+- normipäivä
+- programming
+- Java
+- Solita
+---
+
+##From a Mother to a Software Developer
+
+Clock alarms at 6.15 am. Today I'm lucky and also the children have slept so long. Today is working day, one of my three working days per week. On other days I take care of my three children at home, which is a convenient arrangement for the time being and enabled by my employee, Solita.
+
+After waking up, eating porridge, feeding porridge for my 1,5 years old daughter, making braids for my 6 years old daughter and saying bye-byes also to my 4 years old son and my dear husband, I take my bicycle and pedal to work. Cycling is a wonderful way to switch the context in my head from home matters to work issues. When I arrive to our office, I take my bag and change clothes, like Superman in a phone booth, turning from a mother to a software developer. 
```

Comment:
switch context (the pois)
arrive at
change clothes (pilkku pois) like superman
turning ... into